### PR TITLE
SITL: Callisto model Update

### PR DESCRIPTION
**Pull request by lthall**
This PR updates the Callisto Model to be more realistic.

### CODE REVIEW

File: Tools/autotest/models/Callisto.param [hunk 1]
```diff
@@ -40,15 +55,51 @@ MOT_SPIN_MIN,0.2
...
 PSC_VELZ_P,2.5
+WP_NAVALT_MIN,0
+WP_YAW_BEHAVIOR,1
 WPNAV_ACCEL,250
+WPNAV_ACCEL_Z,100
+WPNAV_JERK,1
+WPNAV_RADIUS,200
+WPNAV_RFND_USE,0
 WPNAV_SPEED,1500
+WPNAV_SPEED_DN,150
```
Comment (rmackay9) [inline Tools/autotest/models/Callisto.param:102]: It looks like at least some of these parameters are the same as the default values.  We should try and avoid adding the defaults to parameter files where possible.

**Comment by rmackay9** [comment]
No big deal but the commit that starts with "Callisto:" should probably start with "Tools:".

File: Tools/autotest/arducopter.py [hunk 2]
```diff
@@ -6606,9 +6606,9 @@ def fly_wind_baro_compensation(self):
             })
             self.reboot_sitl()
             self.set_parameters({
-                "EK3_DRAG_BCOEF_X": 361.000000,
-                "EK3_DRAG_BCOEF_Y": 361.000000,
-                "EK3_DRAG_MCOEF": 0.082000,
+                "EK3_DRAG_BCOEF_X": 91.2220,
```
Comment (lthall) [inline Tools/autotest/arducopter.py:6609]: If this is in the parameter file, do we need it here?
Wouldn't it be more appropriate to be in the parameter file?

**Review by rmackay9 — APPROVED** [review]
This looks OK to me.  I don't have an answer re whether the ekf3-drag parameter should be set as part of the autotest or whether it is sufficient to have them in the Callisto param file.

File: Tools/autotest/arducopter.py (hunk 2 above)
Comment (IamPete1) [reply Tools/autotest/arducopter.py:6609]: I agree, seems odd having it hidden here.

**Review by IamPete1 — APPROVED** [review]
LGTM.

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**